test/js/node/readline/readline.node.test.ts

```diff
@@ -1327,6 +1330,43 @@ describe("readline.Interface", () => {
     assert.strictEqual(getStringWidth("\u0301\u200D\u200E"), 0);
   });
 
+  const testForEventLoopHangs = (fixtureName: string) => async () => {
+    // race a timeout and the subprocess promise
+    const subprocess = Bun.spawn({
+      cmd: ["bun", "run", fixture(fixtureName)],
+    });
```

Comment:
btw i think most tests use `bunExe()` to ensure its using the same bun that ran the test

https://github.com/oven-sh/bun/blob/8064a55a485900a3941eafb8076d55536fda1908/test/js/web/abort/abort.test.ts#L25-L29

Reply:
Fixed